### PR TITLE
Fix#351Update mongo setup in the README

### LINKED ISSUE #351: Update mongo setup in the README
## Description 📹

**Fix the README file to update instructions regarding database setup. Remove references to `db-docker-compose.yml` and provide guidance on obtaining a free MongoDB database from Mongo's website as the preferred setup method.**

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
   Update the README file to:
   - [ ] Remove instructions for using `db-docker-compose.yml`.
   - [ ] Add clear steps for setting up a free MongoDB database using MongoDB Atlas.
   - [ ] Update `connection_string` instructions to reflect the new setup method.
   - [ ] Adjust troubleshooting steps to align with the new database setup.
   - [ ] Verify consistency in other sections of the README referencing the database setup.

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:

   ```bash
   git commit -m "Fix: Update README to reflect new MongoDB setup instructions"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by Amarjeet325**
close: #351

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ If you don't have Rust installed, please go to the [Rust installation page](http
 
 Go to the [Git installation page](https://git-scm.com/downloads) and follow the instructions for your operating system to install Git.
 
-### Install Docker
+### Install Docker(optional)
```
Comment (Marchand-Nicolas) [inline README.md]: Space missing before the "("

File: README.md [hunk 2]
```diff
@@ -60,17 +60,23 @@ cargo build --release
...
+  - Copy the connection string provided, which should look something like this:
+    [Example:]
+     ( mongodb+srv://<username>:<password>@cluster0.mongodb.net/<database>?retryWrites=true&w=majority )
+  - Make sure your IP address is whitelisted in the Atlas security settings.
 
 2. Create `config.toml` file using the `config.template.toml` file.
 Create a `config.toml` file by copying and modifying the `config.template.toml` file. Make sure you update the following fields as required to run the project successfully:
 
-- `connection_string`, this is the string to connect to the database. If the `db-docker-compose.yml` isn't changed the connection string would be: `mongodb://quests:password@localhost:27017`
+- `connection_string`, this is the string to connect to the database.Replace with the MongoDB Atlas connection string here.
```
Comment (Marchand-Nicolas) [inline README.md]: Space missing before "Replace"

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

**Comment by Amarjeet325** [comment]
Hi @Marchand-Nicolas  , Can You recheck my PR , I have  make changes  according to request .

**Review by Marchand-Nicolas — APPROVED** [review]
Perfect lgtm!

**Pull request merged**